go.mod

```diff
@@ -52,6 +52,7 @@ require (
 	github.com/ovh/go-ovh v1.6.0
 	github.com/prometheus/alertmanager v0.28.0
 	github.com/prometheus/client_golang v1.21.0-rc.0
+	github.com/prometheus/client_golang/exp v0.0.0-20250227122456-ad23ad6d5468
```

Comment:
Wanna go ahead and make v0.1.0 release on client_golang? Likely the tag has to be in form of `exp v0.1.0` (https://stackoverflow.com/a/77864755) if you want.

Reply:
Yup totally. I think https://github.com/hashicorp/vault/tags does a similar thing

Reply:
Let's hold off tagging, until I have PR for replacing some of the client code, that way, we can release v0.1.0 with exp we know works.